__integration_tests__/IntegrationUtil.js

```diff
@@ -0,0 +1,8 @@
+const spawnSync = require('child_process').spawnSync;
+const path = require('path');
+
+module.exports.runJest = function runJest(args, dir) {
```

Comment:
I'd probably call this module just `runJest`. I think it makes sense to specify the dir first and args second.